GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.Designer.cs

```diff
@@ -143,6 +146,18 @@ private void InitializeComponent()
             this.chkShowRelativeDate.Text = "Show relative date instead of full date";
             this.chkShowRelativeDate.UseVisualStyleBackColor = true;
             // 
+            // chkShowRepoCurrentBranch
+            // 
+            this.chkShowRepoCurrentBranch.AutoSize = true;
+            this.tlpnlGeneral.SetColumnSpan(this.chkShowRepoCurrentBranch, 2);
+            this.chkShowRepoCurrentBranch.Dock = System.Windows.Forms.DockStyle.Fill;
+            this.chkShowRepoCurrentBranch.Location = new System.Drawing.Point(3, 26);
+            this.chkShowRepoCurrentBranch.Name = "chkShowRepoCurrentBranch";
+            this.chkShowRepoCurrentBranch.Size = new System.Drawing.Size(314, 17);
+            this.chkShowRepoCurrentBranch.TabIndex = 5;
+            this.chkShowRepoCurrentBranch.Text = "Show current branch of repository";
```

Comment:
Too generic, impossible to understand what this changes. Not happy with the following either:
Show current branch in dashboard and dropdown menu

Reply:
I am not happy with it either. Is there a reason for anyone not not show the branch at all? I mean maybe remove this toggle for the user completly?

Reply:
Some want it minimalistic, maybe performance (easier to add an option than to make good performance tests)

Reply:
Would adding a tooltip make sense?

Reply:
Yes
After this, I hope there are no more discussions here...